index.html

```diff
@@ -0,0 +1,144 @@
+            </a>
+
+            <div id="menu-items" class="nav-list">
+            <a class="closebtn" href="">&times;</a>
+                <a class="mobile-nav" href="about.html">About</a>
+                <a class="mobile-nav" href="about.html">Contact</a>
+                <a class="mobile-nav" href="#featured-speakers">PROGRAMS</a>
+                <a class="mobile-nav" href="#">Exhibition</a>
+            </div>
+        </nav>
+    
+      <main id="main">
+    <div class="desk">
+      <p class="before-h12">"Hello! Football Lovers"</p>
+      <h1>FOOTBALL ANNUAL FESTIVAL 2022 </h1>
+      <p class="description-container">
+        FOOTBALL MEETUP Fair is a an annual football exhibition and sales event aimed at projecting the diverse angles of the football industry.
+      </p>
+      <h2>2022.04.30 (Wed) ~ 2ND MAY (Sat) <br><p class="paragraph">@ FT TOWER, Cresecent Avenue, meme, lg, Texas.</p></h2>
+      
+    </div>
+    </div>
+    <div class="cover-program">
+    <section id="program-major">
+        <div class="program-list">
+        <h2>Main Program<hr class="hr"></h2>
+        <ul class="bg-program">
+            <li><img src="./images/program_icon_01.png" alt="Lecture"></li>
+            <li><h3>Lecture</h3></li>
+            <li> <p>demonstrate the visual form of a document or a typeface without relying on mandation</p></li>
+        </ul>
+        <ul class="bg-program">
+            <li><img src="./images/program_icon_02.png" alt="Exhibition"></li>
+            <li><h3>Exhibition</h3></li>
+            <li><p>demonstrate the visual form of a document or a typeface without relying on mandation</p></li>
+        </ul>
+        <ul class="bg-program">
+            <li><img src="./images/program_icon_03.png" alt="Arena"></li>
+            <li><h3>Kids Arena</h3></li>
+            <li><p>demonstrate the visual form of a document or a typeface without relying on mandation</p></li>
+        </ul>
+        <ul class="bg-program program-item">
+            <li><img src="./images/program_icon_04.png" alt="Classes"></li>
+            <li><h3>Classes</h3></li>
+            <li><p>demonstrate the visual form of a document or a typeface without relying on mandation</p></li>
+        </ul>
+        <ul class="bg-program">
+            <li><img src="./images/program_icon_05.png" alt="Restuarant"></li>
+            <li><h3>Restuarant</h3></li>
+            <li><p>demonstrate the visual form of a document or a typeface without relying on mandation</p></li>
+        </ul>
+        <button class="join-btn"><a href="#">Register as a Chef</a></button> 
+        <h5 class="see-whole"><a>SEE THE WHOLE PROGRAM</a></h5>
+        </div>
+      
+    </section>
+    </div>
+
+    <section class="featured-speakers">
+      <h2>Featured Players <hr class="hr"></h2>
+      <div id="featured-speakers">
+       
+      </div>
+      <div>
+        <button class="see-more">More</button>
+      </div>
+    </section>
+
+    </main>
+
+    <div id="partner">
```

Comment:
- [ ] Section tag is used for a stand-alone aspect of the page. What do I mean by stand-alone? I mean when we remove the whole content, the page can still stand and be meaningful. I think that is the case here, the correct semantic to be used here is section tag.